src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/ExternalUsersApiService.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.service
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.Role
+
+@Service
+class ExternalUsersApiService(@Qualifier("externalUsersWebClient") val externalUsersWebClient: WebClient) {
```

Comment:
On the email domain ticket (currently in play) I've created `ExternalUsersEmailDomainApiService` - basically trying to avoid ending up with a very large class. Won't this end up being big if it includes all end points?

Reply:
It will - I can pull it out separately if needed in a later PR - if that's ok